downloader/src/main/kotlin/WorkingTree.kt

```diff
@@ -82,15 +82,15 @@ abstract class WorkingTree(val workingDir: File, val vcsType: VcsType) {
     abstract fun listRemoteTags(): List<String>
 
     /**
-     * Search (symbolic) names of VCS revisions for a match with the given [project] and [version].
+     * Search (symbolic) names of VCS revisions for a match with the given [packageName] and [version].
      *
      * @return The matching VCS revision, never blank.
      * @throws IOException If no or multiple matching revisions are found.
      */
-    fun guessRevisionName(project: String, version: String): String {
+    fun guessRevisionName(packageName: String, version: String): String {
```

Comment:
My initial thinking was to name the parameter `project` here as the function operates on whole repos. And you usually speak about a "project repository", not about a "package repository". Also, a single repo might contain multiple packages / sub-projects, but probably only one (root) project.